data/persist.js

```diff
@@ -51,6 +52,7 @@ export function loadAndPersist( store, reducer, reducerKey, storageKey ) {
 		store.dispatch( {
 			type: 'REDUX_REHYDRATE',
 			payload: persistedState,
+			storageKey,
```

Comment:
Necessary? Would we want a test to confirm it's included in the dispatched action for rehydration?